gradle/databases.gradle

```diff
@@ -124,8 +124,8 @@ ext {
                 mysql : [
                         'db.dialect' : 'org.hibernate.dialect.MySQLDialect',
                         'jdbc.driver': 'com.mysql.cj.jdbc.Driver',
-                        'jdbc.user'  : 'hibernateormtest',
-                        'jdbc.pass'  : 'hibernateormtest',
+                        'jdbc.user'  : 'root',
+                        'jdbc.pass'  : 'chenzhida',
```

Comment:
Create a new user `hibernate_orm_test` with password `hibernateormtest`.   Or do not merge this file.

Reply:
do not commit this file